hack/generate.sh

```diff
@@ -31,10 +31,13 @@ defaulter-gen --input-dirs kubevirt.io/api/core/v1 \
     --output-package kubevirt.io/api/core/v1 \
     --go-header-file ${KUBEVIRT_DIR}/hack/boilerplate/boilerplate.go.txt
 
-openapi-gen --input-dirs kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1,k8s.io/apimachinery/pkg/util/intstr,k8s.io/apimachinery/pkg/api/resource,k8s.io/apimachinery/pkg/apis/meta/v1,k8s.io/apimachinery/pkg/runtime,k8s.io/api/core/v1,k8s.io/apimachinery/pkg/apis/meta/v1,kubevirt.io/api/core/v1,kubevirt.io/api/export/v1alpha1,kubevirt.io/api/export/v1beta1,kubevirt.io/api/snapshot/v1alpha1,kubevirt.io/api/snapshot/v1beta1,kubevirt.io/api/instancetype/v1alpha1,kubevirt.io/api/instancetype/v1alpha2,kubevirt.io/api/instancetype/v1beta1,kubevirt.io/api/pool/v1alpha1,kubevirt.io/api/migrations/v1alpha1,kubevirt.io/api/clone/v1alpha1 \
-    --output-base ${KUBEVIRT_DIR}/staging/src \
-    --output-package kubevirt.io/client-go/api/ \
-    --go-header-file ${KUBEVIRT_DIR}/hack/boilerplate/boilerplate.go.txt >${KUBEVIRT_DIR}/api/api-rule-violations.list
+openapi-gen \
+    --output-dir ${KUBEVIRT_DIR}/staging/src/kubevirt.io/client-go/api/ \
+    --output-pkg kubevirt.io/client-go/api/ \
+    --output-file openapi_generated.go \
+    --report-filename ${KUBEVIRT_DIR}/api/api-rule-violations.list \
+    --go-header-file ${KUBEVIRT_DIR}/hack/boilerplate/boilerplate.go.txt \
+    kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1 k8s.io/apimachinery/pkg/util/intstr k8s.io/apimachinery/pkg/api/resource k8s.io/apimachinery/pkg/apis/meta/v1 k8s.io/apimachinery/pkg/runtime k8s.io/api/core/v1 k8s.io/apimachinery/pkg/apis/meta/v1 kubevirt.io/api/core/v1 kubevirt.io/api/export/v1alpha1 kubevirt.io/api/export/v1beta1 kubevirt.io/api/snapshot/v1alpha1 kubevirt.io/api/snapshot/v1beta1 kubevirt.io/api/instancetype/v1alpha1 kubevirt.io/api/instancetype/v1alpha2 kubevirt.io/api/instancetype/v1beta1 kubevirt.io/api/pool/v1alpha1 kubevirt.io/api/migrations/v1alpha1 kubevirt.io/api/clone/v1alpha1
```

Comment:
nit: Is it possible to have each directory in a line of its own?
Maybe a variable could help here.

Reply:
I will separate each directory into a single line.
I would leave the variable suggestion for a future change, avoiding increasing the scope of this PR, which already contains a lot of changes. Is that OK?

Reply:
Thank you.